packetbeat/docs/packetbeat-options.asciidoc

```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
 Note that limiting documents in this way means that they are no longer correctly
 formatted JSON objects.
 
+[[configuration-tls]]
+=== Capture TLS traffic
+
+++++
+<titleabbrev>TLS</titleabbrev>
+++++
+
+The following settings are specific to the TLS protocol. Here is a sample
+configuration for the `tls` section of the +{beatname_lc}.yml+ config file:
+
+[source,yaml]
+------------------------------------------------------------------------------
+packetbeat.protocols:
+- type: tls
+  send_certificates: true
+  include_raw_certificates: true
+------------------------------------------------------------------------------
+
+==== Configuration options
+
+The `send_certificates` and `include_raw_certificates` settings are useful
```

Comment:
Might help to add another sentence or two here because it's not immediately clear to me how these fields limit the amount of data indexed. Maybe it would help to explain the default behavior when users don't specify these fields. If you think this detail is too much to include in the description, maybe consider including a summary in the intro to this topic.